hoodie-client/src/test/java/com/uber/hoodie/TestMergeOnReadTable.java

```diff
@@ -220,19 +224,21 @@ public void testSimpleInsertAndUpdate() throws Exception {
         assertEquals("Expecting a single commit.", 1, readClient.listCommitsSince("000").size());
         String latestCompactionCommitTime = readClient.latestCommit();
         assertTrue(HoodieTimeline
-            .compareTimestamps("000", latestCompactionCommitTime, HoodieTimeline.LESSER));
+                .compareTimestamps("000", latestCompactionCommitTime, HoodieTimeline.LESSER));
```

Comment:
lets please make sure there are no whitespace or import shuffle changes in the real PR

Reply:
okay will do.